Automattic/jetpack
# Use minified version of videopress-token-bridge

Fixes Automattic/greenhouse#1260

#### Changes proposed in this Pull Request:

This PR allows videopress-token-bridge to be minified instead of including the normal version.

#### Does this pull request change what data or activity we track or use?
No.

#### Testing instructions:

* On a Jetpack plan with VideoPress enabled
* Update projects/packages/assets/src/class-assets.php get_file_url_for_environment by setting $path to $min_path around line 147
* Go to /wp-admin/upload.php and upload a video or use an existing one uploaded to VideoPress
* Edit the video and set it to private
* Create a new post and include the private video
* :white_check_mark: The video should work
* Save and preview
* :white_check_mark: The video should work
* Publish the post
* Go to the post page
* :white_check_mark: The video should work
* Test your post in incognito mode
* :white_check_mark: The private video message should appear
* Go back to /wp-admin/upload.php and make the video public
* Go to your post page
* :white_check_mark: The video should work
* Test your post in incognito mode
* :white_check_mark: The video should work
* CTRL+U on your post page
* Search for token-bridge
* :white_check_mark: The ".min.js" file should be included (in _inc)
